src/frontend/src/layouts/LayoutProps.tsx

```diff
@@ -0,0 +1,8 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+export interface LayoutProps {
```

Comment:
well were extending it to two places right

Reply:
this is for interface segregation adding optionals i dont think is good coding

Reply:
yeah it's kinda overkill for two places using one prop in common

Reply:
isnt it good coding to do interface segregation and reduce code duplication

Reply:
yes, but in this case you're actually adding more code than it would take to just have this prop in both interfaces. If you think we'll add to this interface in the future, or you have some other reason to have the two sets of props extend this one, then we can keep it, but I don't think reducing code duplication is a valid reason here, since it also arguably makes the two sets of props more complicated to look at (especially for newcomers)

Reply:
i mean its like if we ever need to add more props relevant to both of them this makes it easier. its like keeping in mind expandability

Reply:
sure